galley/pkg/config/analysis/diag/level.go

```diff
@@ -68,3 +69,36 @@ func GetUppercaseStringToLevelMap() map[string]Level {
 	}
 	return m
 }
+
+// MessageThreshold is a wrapper around Level to use as cobra command line arguments
+type MessageThreshold struct {
+	Level
+}
+
+// String satisfies interface pflag.Value
```

Comment:
I think this satisfies a different interface.

Reply:
I believe this is correct, according to what I found [here](https://godoc.org/github.com/spf13/pflag#Value).

Reply:
Huh, that's really weird. You're right--I assumed that pflag would just use the Stringer interface instead of specifying a String method. Mea culpa.